<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealtyObject</class>
 <widget class="QWidget" name="RealtyObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>526</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="8">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="13">
      <widget class="QLineEdit" name="leAddress"/>
     </item>
     <item row="0" column="16">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Фактический адрес:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="14">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Тип:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Район:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Тип сделки:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Тек. статус:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="11" colspan="2">
      <widget class="QComboBox" name="cmbStatus"/>
     </item>
     <item row="1" column="6" colspan="2">
      <widget class="QComboBox" name="cmbTradeType"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="cmbRegion"/>
     </item>
     <item row="1" column="13">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="15">
      <widget class="QComboBox" name="cmbType"/>
     </item>
     <item row="1" column="16">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Техпараметры</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Дата созданияя:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spnRooms">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Количество комнат:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dtCreate"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Площадь</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Площадь общая:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnTotalArea">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>м2.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Жилая площадь:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnFloorArea">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>м2.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Площадь кухни:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnKitchenArea">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>м2.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Этажность</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Этаж:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnFloor"/>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Этажность:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnFloors"/>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Материал:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbMaterial">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="chbBalkony">
           <property name="text">
            <string>Балкон</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="chbLoggia">
           <property name="text">
            <string>Лоджия</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Тип квартиры:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbTypeApartament"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Правовой статус</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Собственник:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leOwner"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Док собственности:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbDocProperty">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Контактный телефон:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePhone"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Коммуникации</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Телекоммуникации</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="chbInternet">
            <property name="text">
             <string>Интернет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbPhone">
            <property name="text">
             <string>Телефон</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbCableTV">
            <property name="text">
             <string>Кабельное телевидение</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Бытовые коммуникации</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="chbCentralHeating">
            <property name="text">
             <string>Центральное отопление</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbCentralWater">
            <property name="text">
             <string>Центральное водоснабжение</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chbCentralSewage">
            <property name="text">
             <string>Центральная канализация</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Финансы</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Стоимость и комиссия</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Стоимость объекта:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spnPrice">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Комиссия процент:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Комиссия сумма:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spnAmountCommission">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spnPercentageCommission">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Аренда</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Фото</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Карта</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Описание</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="teDescription"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RealtyObject</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RealtyObject</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>770</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addRegion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1067</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>994</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addTradeType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>1081</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addTypeApartament()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>934</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>addDocumentProperty()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>1061</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RealtyObject</receiver>
   <slot>selectAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1041</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>1073</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveData()</slot>
  <slot>addRegion()</slot>
  <slot>addType()</slot>
  <slot>addTradeType()</slot>
  <slot>addTypeApartament()</slot>
  <slot>addMaterial()</slot>
  <slot>addStatus()</slot>
  <slot>addDocumentProperty()</slot>
  <slot>selectAddress()</slot>
 </slots>
</ui>
